src/content/1.7/webservice/resources/addresses.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Addresses
+---
+
+# Resources for Addresses
+
+### Address
+
+|        Name         |       Format       | Required | Read Only | Max size | Not filterable | Description |
+| :------------------ | :----------------- | :------- | :-------- | :------- | :------------- | :---------- |
+| **id_customer**     | isNullOrUnsignedId |          |           |          |                |             |
+| **id_manufacturer** | isNullOrUnsignedId |          |           |          |                |             |
```

Comment:
I think we might be able to auto generate descriptions for all the ID fields based on their name

`id_manufacturer` => `Manufacturer ID`

Of course they may need additional info, in this address example we might need to add details like `Only use for address related to a manufacturer` But most cases would already be enough I guess

Reply:
I also wonder if we should add a link to the API matching the entity type or is it overkill?

Reply:
> id_manufacturer => Manufacturer ID

I will do it :+1: 

I think the link is overkill right now ^^